src/librustc/util/profiling.rs

```diff
@@ -55,7 +55,7 @@ pub struct SelfProfiler {
 }
 
 impl SelfProfiler {
-    pub fn new() -> Result<SelfProfiler, Box<dyn Error>> {
+    pub fn new(event_filters: &Option<Vec<String>>) -> Result<SelfProfiler, Box<dyn Error>> {
```

Comment:
Were you thinking of making [this change](https://rust-lang.zulipchat.com/#narrow/stream/187831-t-compiler.2Fwg-self-profile/topic/event.20filter.20.2F.20level.20of.20detail/near/163101424) in this PR as well?

Reply:
Eventually, but I'd like to get a perf.rlo run before and after to see if it even makes a difference.

Reply:
Agreed!